src/Form/FeatureType.php

```diff
@@ -36,7 +36,7 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
                 'tom_select_options' => [
                     'create' => true,
                     'options' => $options['centerValues'],
-                    'placeholder' => 'Établissement',
+                    'placeholder' => 'Choisir ou ajouter un établissement',
```

Comment:
could be translated string

Reply:
Actually i'd have to inject the translator interface because it's a tom_select_option so it's not detected as a translation key. What do you think ?